google-cloud-automl/lib/google/cloud/automl/v1beta1/automl_client_helpers.rb

```diff
@@ -0,0 +1,109 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from the file
+# https://github.com/googleapis/googleapis/blob/master/google/cloud/automl/v1beta1/service.proto,
+# and updates to that file get reflected here through a refresh process.
+# For the short term, the refresh process will only be runnable by Google
+# engineers.
+
+
+module Google
+  module Cloud
+    module AutoML
+      module V1beta1
+        class AutoMLClient
+          ##
+          # Gets the latest state of a long-running operation.  Clients can use this
+          # method to poll the operation result at intervals as recommended by the API
+          # service.
+          #
+          # @param name [String]
+          #   The name of the operation resource.
+          # @param options [Google::Gax::CallOptions]
+          #   Overrides the default settings for this call, e.g, timeout,
+          #   retries, etc.
+          # @return [Google::Gax::Operation]
+          # @raise [Google::Gax::GaxError] if the RPC is aborted.
+          # @example
+          #   require "google/cloud/automl"
+          #
+          #   automl_client = Google::Cloud::AutoML::AutoML.new(version: :v1beta1)
+          #   operation_name = "projects/[PROJECT]/locations/[LOCATION]/operations/[OPERATION]"
+          #   operation = automl_client.get_operation(operation_name)
```

Comment:
Drop the parens, both here and in the other method's example.

Reply:
I left the parens in because it matches the code examples for the other methods on the client class.